src/test/java/com/adyen/CheckoutTest.java

```diff
@@ -1288,6 +1301,33 @@ protected PaymentsRequest createEncryptedPaymentsCheckoutRequest() {
         return paymentsRequest;
     }
 
+    /**
+     * Returns a sample PaymentsRequest object with type networkToken and brand visa
+     */
+    protected PaymentsRequest createPaymentsNetworkCheckoutRequest() {
+        PaymentsRequest paymentsRequest = new PaymentsRequest();
+
+        paymentsRequest.setReference("Your order number");
+        paymentsRequest.setAmount(createAmountObject("USD", 1000L));
+        DefaultPaymentMethodDetails defaultPaymentMethodDetails = new DefaultPaymentMethodDetails();
+        defaultPaymentMethodDetails.setType("networkToken");
+        defaultPaymentMethodDetails.setBrand("visa");
+        defaultPaymentMethodDetails.setExpiryMonth("08");
+        defaultPaymentMethodDetails.setExpiryYear("2020");
+        defaultPaymentMethodDetails.setHolderName("CARDHOLDER_NAME");
+        defaultPaymentMethodDetails.setNumber("666666xxxxxx6666");
```

Comment:
Wouldn´t it be more consistent to you a MC masked cardbin?

Reply:
thanks, I replaced it 👍